bin/faucet/src/main.rs

```diff
@@ -56,50 +58,78 @@ pub struct Cli {
     pub command: Command,
 }
 
+#[allow(clippy::large_enum_variant)]
 #[derive(Subcommand)]
 pub enum Command {
     /// Start the faucet server
     Start {
-        #[arg(short, long, value_name = "FILE", default_value = FAUCET_CONFIG_FILE_PATH)]
-        config: PathBuf,
+        /// Endpoint of the faucet in the format `<ip>:<port>`.
+        #[arg(long = "endpoint", value_name = "URL", env = ENV_ENDPOINT)]
+        endpoint: Url,
+
+        /// Node RPC gRPC endpoint in the format `http://<host>[:<port>]`.
+        #[arg(long = "node-url", value_name = "URL", env = ENV_NODE_URL)]
+        node_url: Url,
+
+        /// Timeout for RPC requests in milliseconds.
+        #[arg(long = "timeout", value_name = "MILLISECONDS", default_value_t = 5000, env = ENV_TIMEOUT)]
+        timeout_ms: u64,
+
+        /// Path to the faucet account file.
+        #[arg(long = "account", value_name = "FILE", env = ENV_ACCOUNT_PATH)]
+        faucet_account_path: PathBuf,
+
+        /// Comma-separated list of amounts of asset that should be dispersed on each request.
+        #[arg(long = "asset-amounts", value_name = "U64", env = ENV_ASSET_AMOUNTS, num_args = 1.., value_delimiter = ',', default_value = "100,500,1000")]
+        asset_amounts: Vec<u64>,
+
+        /// Endpoint of the remote transaction prover in the format `<protocol>://<host>[:<port>]`.
+        #[arg(long = "remote-tx-prover-url", value_name = "URL", env = ENV_REMOTE_TX_PROVER_URL)]
+        remote_tx_prover_url: Option<Url>,
+
+        /// The secret to be used by the server to generate the `PoW` seed.
+        #[arg(long = "pow-secret", value_name = "STRING", env = ENV_POW_SECRET)]
+        pow_secret: Option<String>,
+
+        /// Comma-separated list of API keys.
+        #[arg(long = "api-keys", value_name = "STRING", env = ENV_API_KEYS, num_args = 1.., value_delimiter = ',')]
+        api_keys: Vec<String>,
 
         /// Enables the exporting of traces for OpenTelemetry.
         ///
         /// This can be further configured using environment variables as defined in the official
         /// OpenTelemetry documentation. See our operator manual for further details.
-        #[arg(long = "enable-otel", default_value_t = false, env = ENV_ENABLE_OTEL)]
+        #[arg(long = "enable-otel", value_name = "BOOL", default_value_t = false, env = ENV_ENABLE_OTEL)]
         open_telemetry: bool,
     },
 
-    /// Create a new public faucet account and save to the specified file
+    /// Create a new public faucet account and save to the specified file.
     CreateFaucetAccount {
-        #[arg(short, long, value_name = "FILE", default_value = DEFAULT_FAUCET_ACCOUNT_PATH)]
-        output_path: PathBuf,
-        #[arg(short, long)]
+        #[arg(short, long, value_name = "FILE")]
+        output: PathBuf,
+        #[arg(short, long, value_name = "STRING")]
         token_symbol: String,
-        #[arg(short, long)]
+        #[arg(short, long, value_name = "U8")]
```

Comment:
Q: This is a pattern, using `value_name = ALMOST_TYPE_NAME`, but shouldn't it describe the what is to be passed? I.e. `value_name = "PRECISION"` rather than `value_name = "U8"`

Reply:
Yes but in that case we just end up repeating the variable name somewhat so its not more helpful for the user. For example `token_symbol` + `SYMBOL`. Whereas `U8` and `U64` do convey more information to the user for decimals and supply.

The clap docs are really liberal about the canonical usage of value_name:
> Placeholders for the argument’s values in the help message / usage.
>
>These names are cosmetic only, used for help and usage strings only. 

Happy to change if we have opinions on it. Maybe @Mirko-von-Leipzig does too.

Reply:
No strong opinion.

Reply:
No strong opinion either; I tend to just go with whatever I think helps a random person get info from `--help`